packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -389,27 +389,32 @@ module.exports.subscriptionsSubscriptionValidator = isA.object({
 });
 
 // This is support-panel's perspective on a subscription
-module.exports.subscriptionsWebSubscriptionSupportValidator = isA.object({
-  created: isA.number().required(),
-  current_period_end: isA.number().required(),
-  current_period_start: isA.number().required(),
-  plan_changed: isA.alternatives(isA.number(), isA.any().allow(null)),
-  previous_product: isA.alternatives(isA.string(), isA.any().allow(null)),
-  product_name: isA.string().required(),
-  status: isA.string().required(),
-  subscription_id: module.exports.subscriptionsSubscriptionId.required(),
-});
+module.exports.subscriptionsWebSubscriptionSupportValidator = isA
+  .object({
+    created: isA.number().required(),
+    current_period_end: isA.number().required(),
+    current_period_start: isA.number().required(),
+    plan_changed: isA.alternatives(isA.number(), isA.any().allow(null)),
+    previous_product: isA.alternatives(isA.string(), isA.any().allow(null)),
+    product_name: isA.string().required(),
+    status: isA.string().required(),
+    subscription_id: module.exports.subscriptionsSubscriptionId.required(),
+  })
+  .unknown(true);
 
-module.exports.subscriptionsPlaySubscriptionSupportValidator = isA.object({
-  _subscription_type: MozillaSubscriptionTypes.IAP_GOOGLE,
-  auto_renewing: isA.bool().required(),
-  cancel_reason: isA.number().optional(),
-  expiry_time_millis: isA.number().required(),
-  package_name: isA.string().optional(),
-  sku: isA.string().optional(),
-  product_id: isA.string().optional(),
-  product_name: isA.string().required(),
-});
+module.exports.subscriptionsPlaySubscriptionSupportValidator = isA
+  .object({
+    _subscription_type: MozillaSubscriptionTypes.IAP_GOOGLE,
+    auto_renewing: isA.bool().required(),
+    cancel_reason: isA.number().optional(),
+    expiry_time_millis: isA.number().required(),
+    package_name: isA.string().optional(),
+    price_id: isA.string().optional(),
+    product_id: isA.string().optional(),
+    product_name: isA.string().required(),
+    sku: isA.string().optional(),
+  })
+  .unknown(true);
```

Comment:
Why the unknown?

Reply:
Well originally I just added the `price_id` property and that fixed it, however because it's an optional property, the validator test passes regardless of that change. I was trying to find a way to prevent this type of error from ever occurring again.

I noticed in the support panel that only the required fields are displayed and nothing is done with the optional fields (at least for Google IAP subscriptions). So I didn't see the harm in allowing unknown properties.

Also my understanding is that the admin panel work to list subscriptions and create a Support LDAP group will eventually lead us to deprecate and sunset the Support Panel entirely, so preventing our team from spending time fixing bugs in the mean time seems like a good call.

I'm open to just adding the `price_id` property and removing the unknowns if you think that's better?